tests/unit/validation/test_validation_functions.py

```diff
@@ -431,6 +433,81 @@ def test_validate_values__infinite_sigmas(sensor_type, parameter):
         assert not isinstance(error, InfinityError)
 
 
+@pytest.mark.parametrize(
+    ("sensor_type", "parameter", "values", "error_types"),
+    [
+        ("sym_power_sensor", ["p_sigma", "q_sigma"], [NaN, NaN], [InvalidIdError]),
```

Comment:
as discussed, please also add scenarios with 2 components (dw, i'm not trying to put pressure onto you, i'm just adding this comment so that I don't forget about it)